### PR TITLE
Implement slack bot for Coding Coach

### PR DESCRIPTION
**Pull request by mnttnm**
I have only implemented the basic functionality of greeting people when they join the coding coach community on Slack.

Currently, the server is hosted on Heroku and for slack, I am using my personal workspace.
There are certain changes we need to do in slack:
1) Create a new Slack app for our workspace, Name it cc-bot and give proper avatar.
2) Add a bot user and subscribe to events for the app.
3) Install the app for the coding coach workspace.

@crysfel I will need your help w.r.t. deployment.

### CODE REVIEW

**Comment by mnttnm** [comment]
Thanks to @luanjubica, most of the code in this pull request is similar to what he has already implemented. It helped me a lot.
I didn't modify the same pull request as I wanted to learn this from scratch.

File: messages.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      "text": {
+        "type": "mrkdwn",
+        "text": "*Our common channels are:*\n <#CCT9D2TUN> - Please introduce yourself to all :man-raising-hand:\n<#CCRBKGZTR> - General questions, notes and highlights\n<#CCR78Q1LY> - For technical questions mainly\n"
+      }
+    },
+    {
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text":"*Other useful channels:*\n<#CJZQ7GDAP>, <#CKB78JH16>, <#CG42660JK>\n :point_right: <#CL5U9SEJC>, <#CCQDTKZC2> and <#CCPS7NR0R> are for the product related discussions. You can reach :male-technologist:<@DNS22CYQZ> and :male-technologist:<@DNHSLLW69> for questions about the product :point_left:"
```
Comment (crysfel) [inline messages.json]: Please use `@UCQ9GL8UB` and `@UFL4N8ETE`:

```
You can reach :male-technologist:<@UCQ9GL8UB> and :male-technologist:<@UFL4N8ETE> for questions about the product :point_left:
```

**Review by crysfel — CHANGES_REQUESTED** [review]
LGTM! Thanks for contributing! just update some IDs

File: messages.json (hunk 1 above)
Comment (mnttnm) [reply messages.json]: I have made the changes. Could you please review the channel IDs as well?
**#introductions**           -> CCT9D2TUN
**#general**                     -> CCRBKGZTR
**#help**                          -> CCR78Q1LY
**#general-front-end**   -> CJZQ7GDAP
**#general-back-end**   -> CKB78JH16
**#networking**               -> CG42660JK
**#cc-product**               -> CL5U9SEJC
**#cc-product-client**    -> CCQDTKZC2
**#cc-product-api**        -> CCPS7NR0R

**Comment by luanjubica-helius** [comment]
@mnttnm great job. @moshfeu @crysfel sorry I let you guys down, I should have let know earlier.

**Comment by moshfeu** [comment]
No problem @luanjubica-helius. Thanks!

**Review by moshfeu — APPROVED** [review]
Thanks!

**Review by crysfel — APPROVED** [review]
Thanks for contributing!! 🎉

**Comment by mnttnm** [comment]
@crysfel @moshfeu let me know the next steps for this. If anything is needed from my end.

**Comment by moshfeu** [comment]
We've decided to deploy it to heroku. I'll do it asap.

**Pull request merged**